scripts/run-benchmarks.py

```diff
@@ -0,0 +1,200 @@
+import subprocess
+import os
+
+import click
+import cpuinfo
+import requests
+
+BASE_REPO_URL = 'https://github.com/lumol-org/lumol.git'
+
+INDIVIDUAL_BENCH_TEMPLATE = """
+<details><summary>{sha} {title}</summary>
+  <p>\n\n
+\n\n
+```bash\n
+{body}\n\n
+```\n\n
+
+</p></details>
+"""
+
+COMPARISON_TEMPLATE = """
+<details><summary>{sha} {title}</summary>
+  <p>\n\n
+\n\n
+```bash\n
+{body}\n\n
+```\n\n
+
+</p></details>
+"""
+
+
+def _get_environment_variable(name):
+    var = os.environ.get(name, None)
+    if var is None:
+        msg = 'The environment variable {} must be set to access Github API'.format(name)
+        raise Exception(msg)
+
+    return var
+
+
+def check_for_cargo_benchcmp():
+    try:
+        subprocess.check_output('cargo benchcmp --help', shell=True)
```

Comment:
Because if it's not we have to call the commands with a list. For example here `['cargo', 'benchcmp', '--help']` and getting the output is also more painful. This is best to not use `shell=True` for security reasons, I don't see it being a problem here.